callbacks.h

```diff
@@ -18,6 +18,9 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
+#ifndef SCREENTEST_CALLBACKS_H
```

Comment:
An include guard was missing for this header. In this and the other two header files I namespaced the include guards with `SCREENTEST_`.